### PR TITLE
Get the ActiveMQ Helm chart to work successfully on EKS

### PR DESCRIPTION
**Pull request by fabricetriboix**

### CODE REVIEW

**Review by JonHolman — APPROVED** [review]

File: charts/ark-activemq/templates/statefulset.yaml [hunk 1]
```diff
@@ -9,17 +9,18 @@ spec:
   selector:
     matchLabels:
       {{- include "ark-activemq.selectorLabels" . | nindent 6 }}
+  serviceName: {{ include "ark-activemq.fullname" . }}
```
Comment (naveen-armedia) [inline charts/ark-activemq/templates/statefulset.yaml:12]: @fabricetriboix For a statefulset we would require a headless service. Looks service manifest file is not updated with the latest commits. Could you check it.

Comment (fabricetriboix) [reply charts/ark-activemq/templates/statefulset.yaml:12]: Yes, that's right, I forgot that. I will do it now.

**Review by JonHolman — APPROVED** [review]

**Review by ymuwakki — COMMENTED** [review]
{{- fail "At least one of the AMQP, MQTT, Openwire, Websocket or STOMP port must be specified" }}

@fabricetriboix  Instead of a failure why not set a default port for openwire which is what arkcase and alfresco use?  containerPort: {{ .Values.ports.openwire | default 61616 }}

**Comment by fabricetriboix** [comment]
> {{- fail "At least one of the AMQP, MQTT, Openwire, Websocket or STOMP port must be specified" }}
> 
> @fabricetriboix Instead of a failure why not set a default port for openwire which is what arkcase and alfresco use? containerPort: {{ .Values.ports.openwire | default 61616 }}

I would see that as a security issue. Most likely, this port will be left open even if it is not required. ArkCase and Alfresco may use this port at the moment, but we might ditch Alfresco and ArkCase may use a different protocol in the future. 

As a general principle, I prefer to keep things closed by default. Then at instantiation time only what is required can be opened. This is a defensive way of handling such issues. 

In addition, removing a key that is under a dictionary is tricky with Helm.

**Review by ymuwakki — APPROVED** [review]

**Pull request merged**